app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
```

Comment:
It's recommended to add type hints for all function arguments and the return type. In this case, the argument 'cafe' has a missing type hint. This is a code style issue.